regulations/views/sidebar.py

```diff
@@ -8,20 +8,26 @@
 
 class SideBarView(TemplateView):
     """ View for handling the right-side sidebar """
-    def get_template_names(self):
-        return ['regulations/custom-sidebar.html', 'regulations/sidebar.html']
+    template_name = 'regulations/sidebar.html'
+    # contains either class paths or class objects (not instances)
+    components = settings.SIDEBARS
 
     def get_context_data(self, **kwargs):
         context = super(SideBarView, self).get_context_data(**kwargs)
 
         client = api_reader.ApiReader()
 
-        context['sidebars'] = []
-        for class_path in settings.SIDEBARS:
-            module, class_name = class_path.rsplit('.', 1)
-            klass = getattr(import_module(module), class_name)
-            sidebar = klass(context['label_id'], context['version'])
-            context['sidebars'].append(
-                sidebar.full_context(client, self.request))
+        klasses = []
```

Comment:
Why is this named `klasses` rather than `classes`? Or more specifically, is there a reason why it was named `klass` before?